### PR TITLE
Local dev: Skip ingesting if pointing to an RDS DB

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

File: treeherder/config/settings.py [hunk 1]
```diff
@@ -142,6 +143,14 @@
     'default': env.db_url('DATABASE_URL', default=LOCALHOST_MYSQL_HOST),
 }
 
+SKIP_INGESTION = env('SKIP_INGESTION', default=False)
+
+# Block for changes within the context of local development (venv & Docker)
+if not PRODUCTION:
+    # Do not permit ingesting if DATABASE_URL points to a database different than localhost
+    if DATABASES['default']['HOST'] not in ['mysql', '127.0.0.1', 'localhost']:
```
Comment (sarah-clements) [inline treeherder/config/settings.py:151]: Not sure if this is entirely needed since devs should be using the production replica for this.

**Comment by sarah-clements** [comment]
@armenzg were you waiting for @camd to review this?

File: treeherder/config/settings.py (hunk 1 above)
Comment (armenzg) [reply treeherder/config/settings.py:151]: The idea is that if we're running Treeherder locally and DATABASE_URL does not match any valid local DB (`DATABASES['default']['HOST']` is derived from DATABASE_URL) then we should not ingest.

**Comment by armenzg** [comment]
I would appreciate his review. Please merge if satisfied as I will not check Github often.

**Pull request merged**